src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/PersonService.kt

```diff
@@ -46,15 +46,17 @@ class PersonService(
     ConstraintViolationException::class,
   )
 
-  fun processMessage(person: Person, event: String? = null, callback: () -> PersonEntity?) = runBlocking {
+  fun processMessage(person: Person, event: String? = null, callback: (Boolean) -> PersonEntity?) = runBlocking {
```

Comment:
This returns a boolean - it's not very clear what the boolean indicates when reading this code. I know it indicates whether the self match threshold has been met because I have read this PR but I'm sure I will forget that when I am on holiday

Reply:
Improved readability 👍